README.md

```diff
@@ -8,19 +8,21 @@ Saurav, Rob, James, Sree
 
 ### Milestone 1 README
 
-For this project, we will be creating python and R packages that implement k-means clustering from scratch. This will work on any dataset with valid numerical features, and includes fit, predict, and score functions, as well as as elbow and silhouette methods for hyperparameter “k” optimization.
+This package will include python packages that implement k-means clustering from scratch. This will work on any dataset with valid numerical features, and includes fit, predict, and score functions, as well as as elbow and silhouette methods for hyperparameter “k” optimization.
```

Comment:
```Suggestion 
This package will include python packages that implement k-means clustering from scratch. This will work on any dataset with valid numerical features, and includes fit, predict, and cluster_summary functions, as well as as elbow and silhouette methods for hyperparameter “k” optimization.
```